index.html

```diff
@@ -425,7 +425,7 @@ <h5 lang="en">Change language</h5>
         </div>
         <div class="clearfix"></div>
       </div>
-      <p class="text-center">© 2015 SUSE LLC. <span lang="en">All Rights Reserved.</span></p>
+      <p class="text-center">© 2015 SUSE LLC. <span lang="en">All Rights Reserved. </span><span lang="de"><a href="https://de.opensuse.org/Impressum">Impressum</a>.</span></p>
```

Comment:
it is only really relevant for Germany. It certainly could point to en.opensuse.org - and I even created en.opensuse.org/Impressum for it. I didn't know what the <span lang="en"> did though - does it mean visitors with e.g. german browsers won't see that?